Common/TDSQASystemAPI/BL/testpackage/TestPackageLoaderService.cs

```diff
@@ -0,0 +1,127 @@
+        {
+            var testPackage = TestPackageMapper.FromXml(new XmlTextReader(testPackageXmlStream));
+            var validationErrors = new List<ValidationError>();
+
+            //-----------------------------------------------------------------
+            // LOAD TEST PACKAGE CONFIGURATION DATA
+            //-----------------------------------------------------------------
+            itembankConfigurationDataService.CreateClient(testPackage);
+
+            itembankConfigurationDataService.CreateSubject(testPackage);
+
+            var itemStrands = itembankConfigurationDataService.CreateStrands(testPackage);
+
+            itembankConfigurationDataService.CreateStimuli(testPackage);
+
+            var existingItems = itembankConfigurationDataService.CreateItems(testPackage);
+
+            itembankConfigurationDataService.LinkItemToStrands(testPackage, itemStrands);
+
+            itembankConfigurationDataService.LinkItemsToStimuli(testPackage);
+
+            itembankConfigurationDataService.CreateItemProperties(testPackage);
+
+            //-----------------------------------------------------------------
+            // LOAD TEST PACKAGE ADMINISTRATION DATA
+            //-----------------------------------------------------------------
+            itembankAdministrationDataService.SaveTestAdministration(testPackage);
+
+            itembankAdministrationDataService.CreateSetOfAdminSubjects(testPackage);
+
+            itembankAdministrationDataService.CreateAdminStrands(testPackage, itemStrands);
+
+            itembankAdministrationDataService.CreateSetOfAdminItems(testPackage, itemStrands);
+
+            itembankAdministrationDataService.CreateItemMeasurementParameters(testPackage);
+
+            itembankAdministrationDataService.CreateAdminStimuli(testPackage);
+
+            var testForms = itembankAdministrationDataService.CreateTestForms(testPackage);
+
+            itembankAdministrationDataService.CreateTestFormItems(testPackage, testForms);
+
+            itembankAdministrationDataService.CreateAffinityGroups(testPackage);
+
+            //-----------------------------------------------------------------
+            // LOAD TEST PACKAGE SCORING DATA
+            //-----------------------------------------------------------------
+            scoringConfigurationDataService.CreateTest(testPackage);
+
+            scoringConfigurationDataService.CreateGrade(testPackage);
+
+            scoringConfigurationDataService.CreateScoreFeature(testPackage);
+
+            // TODO:  find equivalent of spLoad_ScoringComputationLocation
+
+            scoringConfigurationDataService.CreateComputationRuleParameters(testPackage);
+
+            scoringConfigurationDataService.CreateComputationRuleParameterValue(testPackage);
+
+            // TODO:  find equivalent of spLoad_ScoringConversionTableDesc
+
+            // TODO:  find equivalent of spLoad_ScoringConversionTables

```

Comment:
Scoring Conversions tables were left out of the new test specification.  This was discussed by the team with the conclusion that it was not in use.  We may have to re-visit that decision in the future.

Reply:
Cool - I thought that was the case but wasn't 100% (I couldn't find any reference to this data anywhere either).  I'll remove the TODOs.